components/ReleaseFilter/ReleaseFilter.jsx

```diff
@@ -24,7 +24,8 @@ export default function ReleaseFilter({ releases, onChange }) {
       </label>
       <select
         id="filter"
-        className="input select"
+        className="input select truncate"
+        style={{ maxInlineSize: "300px" }}
```

Comment:
This ensures that the filter doesn't get super long. Will truncate selected option but still shows entire string in the dropdown list.